src/app/examples/todos/todos.reducer.ts

```diff
@@ -74,8 +74,9 @@ export function todosReducer(
             id: uuid(),
             name: action.payload.name,
             done: false
-          }
-        ].concat(state.items)
+          },
+          ...state.items
```

Comment:
The `.concat()` method returns new array it was good enough for the reducer but your solution would work too and be more in line with other code so this change is welcomed.